config/purposes/bioscan.yml

```diff
@@ -20,11 +20,23 @@ LBSN-96 Lysate:
   :input_plate: false
   :type: PlatePurpose::AdditionalInput
   :presenter_class: Presenters::StockPlatePresenter
-  # TODO: Story 568 - Add a specific creator to create and add the randomised control samples.
-  # :creator_class: LabwareCreators::<new bioscan creator>
+  :creator_class: LabwareCreators::StampedPlateAddingRandomisedControls
   :state_changer_class: StateChangers::AutomaticPlateStateChanger
   :work_completion_request_type: 'limber_bioscan_lysate_prep'
   :size: 96
+  :controls:
+    - control_type: pcr positive
+      name_prefix: CONTROL_POS_
+    - control_type: pcr negative
+      name_prefix: CONTROL_NEG_
```

Comment:
Where do we need "CONTROL_POS_" and "CONTROL_NEG_" prefix?
Does the name mean "supplier sample name"? Aren't they prefixed as "CONTROL_" ?

Reply:
its used in the control sample name, sanger sample id and supplier sample name, and was changed to CONTROL_POS_ and CONTROL_NEG_ by R&D.